authorization-service/serverless.yml

```diff
@@ -0,0 +1,28 @@
+service: authorization-service
+frameworkVersion: "3"
+
+useDotenv: true
+plugins:
+  - serverless-dotenv-plugin
+
+provider:
+  name: aws
+  runtime: nodejs16.x
+  region: eu-west-1
+  stage: dev
+
+functions:
+  basicAuthorizer:
+    handler: handler.basicAuthorizer
+    environment:
+      elenapetra: ${env:elenapetra}
```

Comment:
Is this used? As far as I can see, basicAuthorizer takes the username from the event's authorizationToken, not the environment

Reply:
You are right, I've modified the code in order to take the username from lambda environment variable.